apisix/plugins/ext-plugin/init.lua

```diff
@@ -547,6 +533,34 @@ local function create_lrucache()
 end
 
 
+function _M.communicate(conf, ctx)
+    local ok, err, code, body
+    local tries = 0
+    while tries < 3 do
+        tries = tries + 1
+        ok, err, code, body = rpc_call(constants.RPC_HTTP_REQ_CALL, conf, ctx)
+        if ok then
+            if code then
+                return code, body
+            end
+
+            return
+        end
+
+        if not core.string.find(err, "conf token not found") then
```

Comment:
can we use code to determine this, I'm not sure

Reply:
Yes